editoast/src/views/train_schedule/mod.rs

```diff
@@ -371,47 +374,58 @@ async fn get_result(
     Ok(Json(res))
 }
 
-#[derive(Deserialize, IntoParams)]
-struct GetResultsQuery {
+#[derive(Debug, Deserialize, ToSchema, Serialize)]
+struct TrainScheduleResults {
     /// The ID of the path that was used to project the train path
     path_id: Option<i64>,
-    /// The timetable ID
-    timetable_id: i64,
+    train_ids: Vec<i64>,
 }
 
 /// Retrieve the simulation result of multiple train schedules
 #[utoipa::path(
     tag = "train_schedule",
-    params(GetResultsQuery),
+    request_body = inline(TrainScheduleResults),
     responses(
         (status = 200, description = "The train schedule results", body = Vec<SimulationReport>)
     )
 )]
-#[get("/results")]
+#[post("/results")]
 async fn get_results(
     db_pool: Data<DbPool>,
-    query: Query<GetResultsQuery>,
+    request: Json<TrainScheduleResults>,
     core: Data<CoreClient>,
 ) -> Result<Json<Vec<SimulationReport>>> {
-    let query = query.into_inner();
+    use tables::train_schedule::dsl;
+    let body = request.into_inner();
+    let train_ids = body.train_ids;
+    let mut conn = db_pool.get().await?;
+    let schedules = dsl::train_schedule
```

Comment:
```suggestion
    // TODO: replace by ModelV2::batch_retrieve when ready
    let schedules = dsl::train_schedule
```

A remainder for the future me ;)

Reply:
Please apply it 😆

Reply:
Hahaha I forgot sorry 😆